lib/Federation/CloudFederationProviderTalk.php

```diff
@@ -307,10 +307,23 @@ private function roomModified(int $remoteAttendeeId, array $notification): array
 			throw new ShareNotFound(FederationManager::OCM_RESOURCE_NOT_FOUND);
 		}
 
-		if ($notification['changedProperty'] === ARoomModifiedEvent::PROPERTY_AVATAR) {
+		if ($notification['changedProperty'] === ARoomModifiedEvent::PROPERTY_ACTIVE_SINCE) {
+			if ($notification['newValue'] === null) {
+				$this->roomService->resetActiveSince($room);
+			} else {
+				$activeSince = \DateTime::createFromFormat('U', $notification['newValue']);
+				$this->roomService->setActiveSince($room, $activeSince, $room->getCallFlag(), false);
```

Comment:
this will not work, depending on the order the requests arrive or when they arrive in parallel

Reply:
But doesn't all properties have the same problem if they are quickly modified?

Or do you mean regarding sending a request for "active since" and then another one for "in call" both for the same action of modifying the "in call" status? In that case now there is a single request when a call is started which provides the value of both "active since" and "in call".

Reply:
> Or do you mean regarding sending a request for "active since" and then another one for "in call" both for the same action of modifying the "in call" status?

yes

> But doesn't all properties have the same problem if they are quickly modified?

well the case of activesince+callflag is the only one apart from lobby that touches multiple at the same time and they need to come in the same event request

Reply:
As described above now the `ActiveSinceModifiedEvent` includes bot the active since and the call flags.

I noticed a corner case behaviour, though: if some notification is missed for some reason and a newer notification arrives, when the old notification is retried it could wipe the newer value. But that is unrelated to the changes here and more a general problem, so I will open an issue about that.